crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -855,6 +899,30 @@ pub async fn construct_kakarot_test_sequencer() -> TestSequencer {
     TestSequencer::start(SequencerConfig::default(), kakarot_starknet_config()).await
 }
 
+/// Get filepath of a given a compiled contract which is part of the kakarot system
+pub fn get_kakarot_contract_file_path(contract_name: &str) -> PathBuf {
```

Comment:
This seems a little redundant with the `compiled_kakarot_paths` function, could we somehow combine them?

Reply:
Missing update